pxr/base/tf/stringUtils.cpp

```diff
@@ -912,6 +913,72 @@ TfStringify(std::string const& s)
     return s;
 }
 
+template <typename T>
+std::string
+_TfStringifyIntegralImpl(const T& value)
```

Comment:
Should this be just `const T` or `T` in the spirit of this--

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const